docs/installation/mac_and_linux.md

```diff
@@ -13,23 +13,23 @@ Feel free to use another virtual environment manager (e.g. `virtualenvwrapper`)
 * Install Anaconda if you don't already have it.
    * Download the installer [here](https://www.anaconda.com/products/individual), or
    * follow the [installation instructions](https://docs.anaconda.com/anaconda/install/) for your operating system.
-* Create a Python 3.7 (or 3.8) conda environment for opensoundscape: `conda create --name opensoundscape pip python=3.7`
+* Create a Python 3.7.1 (or 3.8) conda environment for opensoundscape: `conda create --name opensoundscape pip python=3.7.1`
 * Activate the environment: `conda activate opensoundscape`
-* Install opensoundscape using pip: `pip install opensoundscape==0.6.1`
+* Install opensoundscape using pip: `pip install opensoundscape==0.6.2`
 * Deactivate the environment when you're done using it: `conda deactivate`
 
 ## Installation via `venv`
 
-Download Python 3.7 (or 3.8) from [this website](https://www.python.org/downloads/).
+Download Python 3.7.1 (or 3.8) from [this website](https://www.python.org/downloads/).
 
 Run the following commands in your bash terminal:
-* Check that you have installed Python 3.7 (or 3.8).\_: `python3 --version`
+* Check that you have installed Python 3.7.1 (or 3.8).\_: `python3 --version`
```

Comment:
Again - should this actually continue to say Python 3.7 - I'm wondering if this is actually confusing listing a specific 3.7.1, when it's compatible with all 3.7.x for x!= 1

Reply:
agreed